lib/prettyprint.js

```diff
@@ -287,11 +287,61 @@ function prettyprintProgram(ast, short) {
             ast.rules.map(prettyprintRule).join('') + '}';
 }
 
+function prettyprintExample(ex) {
+    let args = Object.keys(ex.args);
+    let types = args.map((a) => ex.args[a]);
+
+    let arg_decl = args.length > 0 ? `(${args.map((a, i) => a + ' :' + prettyprintType(types[i])).join(', ')})` : '';
+
+    let value;
+    switch (ex.type) {
+        case 'stream':
+            value = prettyprintStream(ex.value);
+            break;
+        case 'query':
+            value = prettyprintTable(ex.value);
+            break;
+        case 'action':
+            value = prettyprintAction(ex.value);
+            break;
+        case 'program':
+            value = prettyprintProgram(ex.value);
+            break;
+        default:
+            throw new TypeError();
+    }
+    let annotation = `#_[utterances=[${ex.utterances.map((u) => prettyprintValue(u)).join(',')}]]`;
```

Comment:
Is ex.utterances an Array of Ast.Value.String or an Array of JS string?
The grammar suggests the latter, in which case this code is incorrect: `prettyprintValue()` operates on `Ast.Value`

Reply:
It should be JS string. I made a mistake. It's fixed now.